tests/phpunit/src/Commands/CodeStudio/CodeStudioWizardCommandTest.php

```diff
@@ -65,6 +65,8 @@ public function providerTestCommand(): array
                 [$this->getMockedGitLabProject($this->gitLabProjectId)],
                 // Inputs.
                 [
+                    0,
```

Comment:
The test cases are all providing invalid input, and since there are practically no assertions they were overlooked.

This fixes the inputs to at least be valid, but I'm not sure if the test cases are actually testing what they're supposed to...

Reply:
That is what I failed to understand, even when the inputs were wrong the output should not have contained the success messages which was asserted in the last.
I will probably keep this one to be worked upon in future.

Reply:
Yeah, I think this method only returns one line. Ideally you'd be checking the entire output:

```
        $output_strings = $this->getOutputStrings();
```